flink-runtime/src/test/java/org/apache/flink/runtime/messages/webmonitor/JobDetailsTest.java

```diff
@@ -46,7 +46,7 @@ public void testJobDetailsMarshalling() throws JsonProcessingException {
                         9L,
                         JobStatus.RUNNING,
                         8L,
-                        new int[] {1, 3, 3, 7, 4, 2, 7, 3, 3},
+                        new int[] {1, 3, 3, 4, 7, 4, 2, 7, 3, 3},
```

Comment:
I have no idea how to choose the right number. I just choose the random one. If it is specific logic here, please, share it with me.